### DISCUSSION TITLE
why dont more people know about this, now my pc behaves like a linux pc

### DISCUSSION THREAD

**Discussion by bluedinosaur139** [LeDragoX/Win-Debloat-Tools#157] [Q&A] [answered]
### Description

please find some way, like running this whole thing by linus tech tips, gamers nexus, and hardware unboxed

### Reason

well you obviously made this with good intent so why not share it with the masses, i noticed the o and o disclaimer at the bottom which i already use, is it a part of this software?

**Comment by amitkulk123** [comment, marked as answer]
Good software doesn't need to be advertised and there's always risks to using any type of debloater. People already mention similar debloater tools and scripts on subreddits and other forums if you search through the internet and this repo has 3.5k stars which is a pretty decent amount. 

Also a lot of the youtubers you mentioned care more about sponsorships and so they tend to recommend whatever makes them more money rather than what is necessarily good.

**Comment by LeDragoX** [comment]
Converting to discussion and closing.
